api/v1alpha1/contour_types.go

```diff
@@ -51,6 +51,33 @@ type ContourSpec struct {
 	// +kubebuilder:default=2
 	// +kubebuilder:validation:Minimum=0
 	Replicas int32 `json:"replicas,omitempty"`
+
+	// namespace defines the schema of a Contour namespace.
+	// See each field for additional details.
```

Comment:
What should happen if multiple Contour resources specify the same namespace?

Reply:
@jpeach I would expect multiple instances of Contour to run in said namespace.